include/z64.h

```diff
@@ -66,7 +66,7 @@
 typedef struct {
     /* 0x0 */ s16 priority; // Lower means higher priority. -1 means it ignores priority
     /* 0x2 */ s16 length;
-    /* 0x4 */ s16 unk4;
+    /* 0x4 */ s16 csCamSceneDataId; // Index of CsCameraEntry to use. Negative indices use sGlobalCamDataSettings. Positive indices index CsCameraEntry from scene
```

Comment:
In that case, it would index the 0th entry of `CsCameraEntry`, i.e. the first entry. Maybe I should clarify that it's 0+ instead of just positive numbers

Reply:
-1 is also a special case but not sure if I should mention that in this comment here or leave it for where -1 is a special case inside the function `Camera_ChangeDataIdx`